cmd/snap-seccomp/syscalls/syscalls.go

```diff
@@ -36,6 +36,8 @@ var SeccompSyscalls = []string{
 	"arch_prctl",
 	"arm_fadvise64_64",
 	"arm_sync_file_range",
+	"atomic_barrier",
```

Comment:
The comment above this says: ```
Generated using arch-syscall-dump test tool from libseccomp tree, git
// revision bf747eb21e428c2b3ead6ebcca27951b681963a0.
var SeccompSyscalls = []string{
```
do we need to update the git revision of this comment?

Reply:
yea probably good idea